pom.xml

```diff
@@ -64,6 +64,30 @@
             <groupId>io.micrometer</groupId>
             <artifactId>micrometer-core</artifactId>
         </dependency>
+<!--        <dependency>-->
```

Comment:
Any specific reason for keeping this commented code snippet?

Reply:
The Dependency of Mockito is used to create mocks and stubs. but the spring-boot-starter-test dependency is sufficient to.